service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/transport/amqps/AmqpConnectionHandler.java

```diff
@@ -130,6 +177,15 @@ public void onConnectionBound(Event event)
         }
     }
 
+    @Override
+    public void onConnectionInit(Event event)
+    {
+        Connection conn = event.getConnection();
+        conn.setHostname(hostName);
+        log.debug("Opening connection for amqp cloud to device message sender");
```

Comment:
Is this debug statement correct?

Reply:
It wasn't correct, but it has since been fixed